cookbook/specs/spec_add_sqdsubgraph.json

```diff
@@ -96,11 +96,43 @@
                             "api_interface": "rest",
                             "internal_path": "",
                             "type": "POST",
-                            "add_on": "convex"
+                            "add_on": "compound-v3"
                         },
                         "apis": [
                             {
-                                "name": "/subgraphs/name/convex",
+                                "name": "/subgraphs/name/compound-v3",
```

Comment:
is covex not supported anymore?

Reply:
Yes, we are going with Aave and compound for now